kubernetes/metasploit/templates/tests/test-connection.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: "{{ include "metasploit.fullname" . }}-test-connection"
+  labels:
+    {{- include "metasploit.labels" . | nindent 4 }}
+  annotations:
+    "helm.sh/hook": test-success
+spec:
+  containers:
+    - name: wget
+      image: busybox
+      command: ['wget']
+      args: ['{{ include "metasploit.fullname" . }}:{{ .Values.service.port }}']
```

Comment:
I don't believe port 80 will have anything running on it by default, just confirming if this test-connection functionality useful for msfconsole 👀